test/testmachinery/shoots/logging/seed_logging_stack.go

```diff
@@ -30,6 +30,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
```

Comment:
Can't we use logr as in the other tests? Afaik, "k8s.io/klog/v2" is only used by gardener-apiserver currently. Ref https://github.com/gardener/gardener/blob/4507f53471f8f982e1351af499a5ed2804ee65d4/docs/development/logging.md#L39

Reply:
Comply with the decision.